programs/uxd/src/state/mercurial_vault_depository.rs

```diff
@@ -4,25 +4,29 @@ use fixed::types::I80F48;
 use crate::error::UxdError;
 
 // Total should be 900 bytes
-pub const MERCURIAL_VAULT_RESERVED_SPACE: usize = 645;
+pub const MERCURIAL_VAULT_RESERVED_SPACE: usize = 588;
 pub const MERCURIAL_VAULT_DEPOSITORY_SPACE: usize = 8
```

Comment:
nit: we could create a unit test to make this computation safer for future edits:
test_credix_lp_depository.rs:
https://github.com/UXDProtocol/uxd-program/commit/ebf8d8814a2e731f9982e334dd05e867db783b67#diff-a8e9614f5be09945d9ff5b833f172789b73aa62cc04aeff55fed158ab5ab0688

Reply:
I think there is a macro somewhere that does checks and all, went looking for it on the Jet-v2 but cannot fight it back... Worth investigating